pwa/src/components/endpoints/form.tsx

```diff
@@ -0,0 +1,241 @@
+import * as React from "react";
```

Comment:
Overkoepelende feedback voor dit component: volgens mij heeft die enorm veel overlap met `applicationForm`. Met zoveel overlap is het misschien interessant om te kijken hoe we de generieke onderdelen los kunnen trekken zodat we deze kunnen hergebruiken in beide plekken.

Nu krijgen we bijvoorbeeld het probleem dat wanneer we aanpassingen willen maken in de `getItem` functie, dat we die op meerdere plekken moeten updaten.